data_quality/main.py

```diff
@@ -83,14 +71,41 @@ def batch_handler(instance):
 @cli.command()
 @click.argument('config_filepath')
 def deploy(config_filepath):
-
     """Deploy data sources for a Spend Publishing Dashboard instance."""
 
-    with io.open(config_filepath, mode='rt', encoding='utf-8') as f:
-        config = json.loads(f.read())
-
+    config = helpers.load_json_config(config_filepath)
+    config['data_dir'] = helpers.resolve_relative_path(config_filepath, config['data_dir'])
     deployer = tasks.Deploy(config)
     deployer.run()
 
+
+@cli.command()
+@click.argument('generator_class')
+@click.argument('endpoint')
+@click.argument('config_filepath', type=click.Path(exists=True))
```

Comment:
@georgiana-b for all `dq` commands, `config_filepath` should be in the same location from all positional arguments that a command takes. This is not the case with this new command. However, based on other comments on these PRs, `config_filepath` could become an optional argument if we always read a default config.